modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1217,7 +1217,8 @@ public int getOneToManyObjectEntriesCount(
 	@Override
 	@Transactional(propagation = Propagation.REQUIRED)
 	public ObjectEntry getOrAddIncompleteObjectEntry(
-			String externalReferenceCode, long userId, long objectDefinitionId)
+			String externalReferenceCode, long userId, long groupId,
+			long objectDefinitionId)
```

Comment:
Please sort.
externalReferenceCode > groupId > userId > objectDefinitionId
See ObjectEntry in service.xml